docs/content/drafts/Tooltip.mdx

```diff
@@ -0,0 +1,80 @@
+---
+componentId: tooltip
+title: Tooltip
+status: Alpha
+---
+
+import data from '../../../src/drafts/Tooltip/Tooltip.docs.json'
+
+The Tooltip component is used to add context to interactive elements on the page.
+
+**_⚠️ Usage warning! ⚠️_**
+
+Tooltips as a UI pattern should be our last resort for conveying information because it is hidden by default and often with zero or little visual indicator of its existence.
+
+Before adding a tooltip, please consider: Is this information essential and necessary? Can the UI be made clearer? Can the information be shown on the page by default? See [Tooltip alternatives](https://primer.style/design/accessibility/tooltip-alternatives) for more accessible alternatives.
+
+Tooltip should only be used on an element that is interactive such as a button or a link.
+
+## Examples
+
+### Default (As a description type)
+
+```jsx live
+<Tooltip text="This change cannot be undone.">
+  <Button>Delete</Button>
+</Tooltip>
+```
+
+### As a label type
+
```

Comment:
Ditto for the title here, something like "for primary label"? 

As description is the default, we should add a small text here for when you should reach out for a label. Is it only when there is no visual text in the element?

If yes, we can say something along the lines of "If there is no visual text content in the element and the Tooltip serves as a primary label, add `type=label` to the Tooltip.

Reply:
commented here https://github.com/primer/react/pull/3394#discussion_r1326663240 😊

Reply:
feel free to suggest changes if there is any part that you don't feel comfortable or still confusing!